src/tests/testData/sampleData.js

```diff
@@ -2,16 +2,33 @@ export const multiRequest = {
   source: 'Nigeria',
   destination: 'Abuja, USA, POLAND',
   tripType: 'multi-city',
-  travelDate: '2038-01-19 03:14:07 ',
-  returnDate: '2038-01-19 03:14:07',
+  travelDate: Date.now(),
+  returnDate: Date.now(),
   reason: 'reason',
   accommodation: 'accommodation'
 };
 
 export const missingRequiredField = {
```

Comment:
travelDate is a required field, i think you should check is entered

Reply:
Ok